src/views/index_view.js

```diff
@@ -31,6 +31,16 @@ export default BaseView.extend({
     // up arrow and shift+tab move up
     } else if (data.key === 'ArrowUp' || (data.key === 'Tab' && data.shiftKey)) {
       this._selectNextItem(-1);
+    // enter triggers a navigation
+    } else if (data.key === 'Enter') {
+      let selectedItem = this.query('li.selected');
+      if (selectedItem) {
+        // trick the element into navigating
+        // TODO: use JS events, not DOM events, for signaling
+        var fakeClick = new CustomEvent('mousedown');
+        fakeClick.which = 1;
```

Comment:
The `mousedown` handlers check that it was a left mouse key (`which = 1`), so we have to mock that out here.

It's certainly less than ideal to handle keys by overloading click handlers, but it does reliably propagate the right signals back and forth, so hopefully we can merge as-is and rethink the API soon.